distributed/worker.py

```diff
@@ -984,20 +982,13 @@ def status(self, value):
             self.handle_stimulus(UnpauseEvent(stimulus_id=stimulus_id))
 
     def _send_worker_status_change(self, stimulus_id: str) -> None:
-        if (
-            self.batched_stream
-            and self.batched_stream.comm
-            and not self.batched_stream.comm.closed()
-        ):
-            self.batched_stream.send(
-                {
-                    "op": "worker-status-change",
-                    "status": self._status.name,
-                    "stimulus_id": stimulus_id,
-                },
-            )
-        elif self._status != Status.closed:
-            self.loop.call_later(0.05, self._send_worker_status_change, stimulus_id)
```

Comment:
This retry will now be lost. I imagine this case occurs if the worker's status changes before it's connected to the scheduler. I'm not sure if this is important.

As part of https://github.com/dask/distributed/issues/6389, we could pretty easily make `BatchedSend` support queuing messages before it's been started.

Reply:
This used to be necessary when I implemented it - the WorkerState on the scheduler side would never emerge out of initialising state. Now it is not needed anymore.